webui/src/pages/auth/policies/index.jsx

```diff
@@ -43,6 +44,8 @@ const PoliciesContainer = () => {
     if (error) return <Error error={error}/>;
     if (loading) return <Loading/>;
 
+    const showRBACWarning = useLoginContext()?.RBAC === 'simplified';
```

Comment:
It's subtle, but this is still a conditional use of a hook. This needs to move above line 44.

Reply:
Of course.  Done.